lib/animas/animasSimulator.js

```diff
@@ -167,24 +167,25 @@ exports.make = function(config){
       }
 
       if(currStatus !== null) {
-        if(currStatus.status === 'suspended') {
-          if (event.rate === 0) {
+        if (event.rate === 0 ) {
```

Comment:
I took me several reads of this if/else block to get it. I think a comment might be useful, perhaps to the effect of "categorize the basal we're processing as a suspend basal if it's rate is 0 and its start falls within the suspend event we have on deck". (Did I get that right?)